python/flex.py

```diff
@@ -23,33 +23,32 @@
     name = item.GetName()
     print("Loaded Assembly: {0}".format(name))
 
-# Import the API
+# Import the API (has to be after the loaded assemblies are printed)
```

Comment:
It just needs to be after the clr.AddReference calls.  

I'm only printing out loaded assemblies for debugging purposes.

Reply:
fixed in latest commit